.travis.yml

```diff
@@ -0,0 +1,21 @@
+matrix:
+  include:
+    - os: osx
+  fast_finish: true
+
+install:
+  - brew install sox
+  - brew install ffmpeg
+  - brew cask install mactex
+  - pip2 install -r requirements.txt
+
```

Comment:
Could we please insert:
```
    - pip install flake8  # pytest  # add another testing frameworks later
before_script:
    # stop the build if there are Python syntax errors or undefined names
    - flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
    # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
    - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Reply:
sure, let's first merge the bare bones CI support and then we can build on it, add `flake8`, real tests, etc. Does it sound good?

Reply:
Cool...  We would also want to start parallel testing on Python 3 in __allow_failures__ mode as in https://github.com/andrix/python-snappy/pull/66/files#diff-354f30a63fb0907d4ad57269548329e3R4